app/src/main/res/xml/autofill_service_configuration.xml

```diff
@@ -220,12 +220,6 @@
     <compatibility-package
         android:name="org.adblockplus.browser.beta"
         android:maxLongVersionCode="10000000000" />
-    <compatibility-package
-        android:name="org.bromite.bromite"
-        android:maxLongVersionCode="10000000000" />
-    <compatibility-package
-        android:name="org.bromite.chromium"
-        android:maxLongVersionCode="10000000000" />
```

Comment:
I'm not comfortable removing bromite in the context of this PR. Although we did find where native autofill support was added to bromite, we don't want to break autofill for versions prior to the native implementation being added. 

I would prefer to declare the `maxLongVersionCode` of the release prior to native autofill support being added so that compat mode does not ignore the package entirely. That change would need to be in a separate PR though.

I'll notify the team and have this added to our backlog. Thank you for bringing it to our attention.

Reply:
Bromite has over 600 known security issues.

Reply:
This was a nice to have IMO but it isn't the focus here so it isn't available anymore, I wanna help provide access to passkeys to these communities and while it would be nice to have a slight hint to not use Bromite, if it makes it harder to get these things going, then this could be suggested in another PR and such.

The main list trusts other browsers I don't think most people here would trust anyway. (e.g. Yandex, Chrome, Edge...) I'm surprised Vanadium is there... gives a reason to support Mulch as well.